src/paxos.rs

```diff
@@ -557,27 +560,12 @@ where
                     .filter_map(|p| p.as_ref())
                     .filter(|p| p.pid != leader_pid);
                 for PromiseMetaData {
-                    n: promise_n,
-                    la: promise_la,
+                    n: _promise_n,
+                    la: _promise_la,
                     pid,
                 } in promised_followers
                 {
-                    let msg = if cfg!(feature = "max_accsync")
-                        && (promise_n, promise_la) == (max_promise_n, max_la)
-                    {
-                        Message::with(
-                            self.pid,
-                            *pid,
-                            PaxosMsg::AcceptSync(max_promise_acc_sync.clone()),
-                        )
-                    } else if cfg!(feature = "max_accsync")
-                        && promise_n == max_promise_n
-                        && promise_la < max_la
-                    {
-                        let sfx = self.storage.get_suffix(*promise_la);
-                        let acc_sync = AcceptSync::with(self.n_leader.clone(), sfx, *promise_la);
-                        Message::with(self.pid, *pid, PaxosMsg::AcceptSync(acc_sync))
-                    } else {
```

Comment:
These checks MUST still be checked i.e. we must still do `if (promise_n, promise_la) == (max_promise_n, max_la) { ... } else { ... }`